docs/devices/ZB-TTS01.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Easyiot ZB-TTS01 control via MQTT"
+description: "Integrate your ZB-TTS01 via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
+addedAt: 2024-01-11T16:23:11Z
+pageClass: device-page
+---
+
+<!-- !!!! -->
+<!-- ATTENTION: This file is auto-generated through docgen! -->
+<!-- You can only edit the "Notes"-Section between the two comment lines "Notes BEGIN" and "Notes END". -->
+<!-- Do not use h1 or h2 heading within "## Notes"-Section. -->
+<!-- !!!! -->
+
+# Easyiot
+
+|     |     |
+|-----|-----|
+| Model | ZB-TTS01 |
+| Vendor  | [Easyiot](/supported-devices/#v=Easyiot)  |
+| Description | This is a text-to-speech controller that can convert GB2312 Chinese, letters, and numbers into Chinese speech. |
+| Exposes | send tts, last received status, linkquality |
+| Picture | ![ZB-TTS01](https://www.zigbee2mqtt.io/images/devices/ZB-TTS01.jpg) |
+
+
+<!-- Notes BEGIN: You can edit here. Add "## Notes" headline if not already present. -->
+## Notes
+## 1 Protocol Overview
```

Comment:
I think these details shouldn't be here, a z2m users does not have to deal with all of this since it is abstracted away.